NAMESPACE

```diff
@@ -17,5 +17,5 @@ export(rebuild_cmdstan)
 export(register_knitr_engine)
 export(set_cmdstan_path)
 export(set_num_threads)
+export(write_stan_file)
 export(write_stan_json)
-export(write_stan_tempfile)
```

Comment:
Wouldnt there be an error if the deprecated function is not exported?

Reply:
Yes, good catch! I meant to still export it. Should be fixed now.